dictcc.el

```diff
@@ -203,8 +213,7 @@ At the moment they are of the form `<tr id='trXXX'></tr>'."
 
 (defun dictcc--candidate (pair)
   "Generate the candidate pair for a PAIR of translations."
-  (let* ((format-string (format "%%-%ds -- %%%ds"
-                                dictcc-candidate-width
+  (let* ((format-string (format "%%-%ds   %%s"
```

Comment:
Sorry, this is personal preference.  I thought that the translations are too far apart because the destination language was aligned to the right.  I changed the alignment to the left and removed the ---, like it is on the homepage.  Feel free to ignore this change.